docs/source/development.rst

```diff
@@ -0,0 +1,46 @@
+Development
+===========
+
+In order to work on ``datajudge``, you can create development conda environment as follows:
+
+::
+
+    git clone https://github.com/Quantco/datajudge
+    cd datajudge
+    mamba env create
+    conda activate datajudge
+    pip install --no-build-isolation --disable-pip-version-check -e .
+
+Unit tests can be run by executing
+
+::
+
+   pytest tests/unit
+
+Integration tests are run against a specific backend at a time. As of now, we provide helper
+scripts to spin up either a postgres or mssql backend.
+
+To run integration tests against postgres, first start a docker container with a postgres database:
+
+::
+
+   ./start_postgres.sh
+
+Once this is running, you may execute integration tests as follows:
+
+::
+
+   pytest tests/integration --backend=postgres
+
+Analogously, for mssql, run
+
+::
+
+   ./start_mssql.sh
+
+and
+
+::
+
+   pytest tests/integration --backend=mssql-freetds
```

Comment:
Shouldn't the standard example be `--backend=mssql`? Without the `-freetds` and then adding a note for ARM users?

Reply:
The original motivation was for it to run on Apple laptops. I haven't tried the `freetds` driver on Windows, probably should work.

Reply:
I added a comment.